components/sysutils/mlocate/Makefile

```diff
@@ -23,11 +23,12 @@
 # Copyright (c) 2013, Oracle and/or its affiliates. All rights reserved.
 #
 
+BUILD_BITS= 64
```

Comment:
Please remove this line.  It is default.

Reply:
done